### ISSUE TITLE
Hail Storm cannot be picked up if WELL is re-entered before picking it up

### ISSUE THREAD

**Issue by pehj42** [TheLegendOfMataNui/game-issues#278]
In the room where Kopaka uses the x-ray mask to hop on invisible platforms, the elemental glyph can't be picked up and doesn't spin. It acts like I've already picked it up, but I haven't.

Rebuilt on Windows 10

**Comment by pehj42** [comment]
I theorize it might have to do with getting to the door too early. In the Pohatu level, after the Kane-Ra Bull run,  it dropped be off in some forked tunnels. One of the those tunnels, in another loaded area, leads to a door that can only be opened from one side. going to this door caused my game to crash. Every time I opened that save file it would immediately crash. I started a new game and ignored that part in order to continue.

In Kopaka's level I did something similar where I encountered a door that could only be opened from one side. After I found the Ko-Kini, I went through a passageway that lead outside. there were a couple bridges and a platform with four rahi spawners on it, inclucing a Mauka. In this area I found a door that couldn't be opened with the same symbol on it. I figured I'd gone the wrong way again so I went back to the Kini and went through the other passageway and eventually made it to the invisible platform room.

Hope this helps!
